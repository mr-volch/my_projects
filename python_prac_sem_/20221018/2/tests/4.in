:root x y z j sqrt((x+y+z+j)**2)
root 3 1 2 4
:isalpha s d a i (a+s+d+i).isalpha()
:none a print(a)
isalpha "ЖЖЖ" "AAAА" "С" "2"
isalpha "s" "d" "a" "i"
none "l"
quit "{} <first - second> {}"
