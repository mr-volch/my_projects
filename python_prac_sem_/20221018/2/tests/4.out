10.0
False
True
l
None
4 <first - second> 8
